assets/templates/dataset-filter/filter-overview.tmpl

```diff
@@ -19,12 +19,12 @@
                   {{ template "partials/latest-release-alert" . }}
 	            {{ end }}
                     <div class="col col--md-47 col--lg-59 margin-top--2 margin-bottom--4 background--gallery">
-                        <ul class="list--neutral filter-overview">
-                            <li class="margin-left--0 padding-bottom--2 padding-top--0 padding-right--2 width-lg--56">
+                        <ul class="list--neutral filter-overview ">
+                            <li class="font-size--18 line-height--32 margin-left--0 padding-bottom--2 padding-top--0 padding-right--2 width-lg--56">
```

Comment:
Because you have set `adjust-font-size--18` in the `div` a few levels above (line 13), you technically don't need to set the font-size on each `li` element inside it. This is because this class in Sixteens cascades its rules to any elements inside it. You can see how the class works in Sixteens, `_utilities.scss`, line 641

In this case, we set `font-size: 18px` on line 13, and each child element for that will inherit that font-size. CSS is underpinned by three core ideas: cascade, inheritance, and specificity. You can read more about them here https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Cascade_and_inheritance